setup.cfg

```diff
@@ -9,6 +9,11 @@ show-source = True
 doctests = False
 statistics = False
 
+# darglint configuration
```

Comment:
@sobolevn I am not sure it's best place to store `darglint` configuration, however it's enforced by `NIP324`:

```
NIP324 File setup.cfg: section [flake8] has some missing key/value pairs. Use this:

[flake8]
docstring_style = numpy
strictness = long
```

Reply:
I have switched to this config format due to `--isolated` feature: https://github.com/terrencepreilly/darglint/issues/55
Otherwise, `darglint` was always reading its config. It caused several issues with other tools.